TwitchLib.PubSub/Models/Responses/Message.cs

```diff
@@ -20,10 +20,10 @@ public Message(string jsonStr)
             switch (Topic?.Split('.')[0])
             {
                 case "chat_moderator_actions":
-                    MessageData = new ChatModeratorActions(encodedJsonMessage);
+                    MessageData = new ChatModeratorAction(encodedJsonMessage);
```

Comment:
I actually liked the change you had here, where the class was changed to `ChatModeratorActions` from `ChatModeratorAction` because the twitch provided identifier is `chat_moderator_actions`, so the new name would make sense.

Reply:
I personally really disagree with you. Twitch just gives back data with a "chat_moderator_actions" tag, indicating that it it's a message which can be 'any of the chat moderation actions'. However, it will always be 1 single action. Not multiple. So changing the name to Actions will describe the class incorrectly. It's not a class which contains multiple actions. No. It's always a single action.